Client/src/components/contanier/student/Profile/index.js

```diff
@@ -1,10 +1,24 @@
 import React, { Component } from 'react'
+import TextCard from '../../../common/TextCard'
+import './index.css';
 
 export default class Profile extends Component {
+  state = {
+    level: 6,
+  }
+
   render() {
+    const label = ['Name :', 'Phone Number :', 'Amideast ID :']
+    const data = ['Abdallah Amamr', '0597185554', '59763']
     return (
-      <div>
-        from Profile
+      <div className='profile'>
+        <h1 className='profile-titel'>Profile</h1>
+        <h2 className='profile-level' >Level {this.state.level}</h2>
+        {label.map((item, index) => {
+          return (
+            <TextCard data={data[index]} label={item} key={index} />
```

Comment:
### Look at the annoying difference in font size
![React App 4-22-2019 4-03-53 PM](https://user-images.githubusercontent.com/39484730/56501662-3be51480-6518-11e9-89a5-b32887f4267f.png)

Reply:
I make some edits you can see it 
and hope you can give me some suggestions

Reply:
![React App 4-22-2019 5-26-59 PM](https://user-images.githubusercontent.com/39484730/56505311-fdeded80-6523-11e9-9495-3688b2d7a1d5.png)


## You Can :
1- Change the font family.
1-  Put the label and data in the same line in all screens

Reply:
I changed the font family